addons/mail/static/tests/tours/discuss_public_tour.js

```diff
@@ -57,7 +58,9 @@ registry.category("web_tour.tours").add("mail/static/tests/tours/discuss_public_
         {
             content: "Check the earlier provided attachment is listed",
             trigger: '.o-mail-attachment-card[title="text.txt"]',
-            run() {},
+            async run() {
+                await waitUntil('.o-mail-attachment-card[title="text.txt"]');
```

Comment:
I don't understand, for me they are all unnecessary because it's already what the "trigger" does, waiting for the element to be present.